### PR TITLE
Adding CDS Reef blog post

### PR DESCRIPTION
**Pull request by Thingee**
Signed-off-by: Mike Perez <thingee@gmail.com>

### CODE REVIEW

File: src/en/community/events/2022/ceph-developer-summit-reef/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Ceph Developer Summit - Reef
 date: 2022-04-12
 end: 2022-04-16
```
Comment (jdurgin) [inline src/en/community/events/2022/ceph-developer-summit-reef/index.md:4]: end 04-22

File: src/en/news/blog/2022/ceph-developer-summit-reef/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+v.17.1.0](https://ceph.io/en/news/blog/2022/newsletter-march/#ceph-quincy-release-canadidate-v17.1.0-is-now-available),
+plans for the next release, Reef, are underway with the next Ceph Developer
+Summit!
+
+Typically we would hold the summit at
+[Cephalocon](https://events.linuxfoundation.org/cephalocon/), but the
+conference was postponed to July 11-13th. In the interest of keeping momentum
+going, there will be a [virtual Ceph Developer
+summit](https://ceph.io/en/community/events/2022/ceph-developer-summit-reef/)
+April 12-16th.
```
Comment (jdurgin) [inline src/en/news/blog/2022/ceph-developer-summit-reef/index.md:21]: 22nd

**Review by jdurgin — APPROVED** [review]
looks good! just the end date should be updated

**Comment by Thingee** [comment]
Good catch!

**Pull request merged**